share/functions/__fish_config_interactive.fish

```diff
@@ -24,14 +24,28 @@ function __fish_config_interactive -d "Initializations that should be performed
 
     #
     # If we are starting up for the first time, set various defaults
-    #
-    if not set -q __fish_init_1_50_0
+    #    
+    if not set -q __fish_init_2_39_8 # bump this to 2_40_0 when rolling release if anything changes after 9/10/2016
```

Comment:
I want folks to pound on it with 0-8 colors, bright colors, bold normal colors, and #RRBBGG colors. So I did this to blast away color settings. It doesn't need to be kept in for the real release.